.buildkite/scripts/pre-install-command.ps1

```diff
@@ -0,0 +1,25 @@
+Write-Host "-- Fixing CRLF in git checkout --"
+# Forcing to checkout again all the files with a correct autocrlf.
+# Doing this here because we cannot set git clone options before.
+git config core.autocrlf input
+git rm --quiet --cached -r .
+git reset --quiet --hard
+
+Write-Host "-- Install golang --"
+choco install -y golang --version $env:SETUP_GOLANG_VERSION
+$env:ChocolateyInstall = Convert-Path "$((Get-Command choco).Path)\..\.."
+Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
+refreshenv
+go version
+go env
+
+Write-Host "-- Install go-junit-report --"
+go install github.com/jstemmer/go-junit-report/v2@latest
+
+Write-Host "-- Run test, prepare junit-xml --"
+go mod download -x
+go test -v 2>&1 ./... | go-junit-report > "tests-report-win-unicode.xml"
```

Comment:
In the current Jenkinsfile, tests in windows are executed using `goTestJunit`

https://github.com/elastic/package-registry/blob/main/.ci/Jenkinsfile#LL101C23-L101C23

In that function (https://github.com/elastic/apm-pipeline-library/blob/main/vars/goTestJUnit.groovy#L31) , it is used gotestsum

Could it be used the same ? 

You would need to install `gotestsum` as it is done in the `withMageEnv`
https://github.com/elastic/apm-pipeline-library/blob/main/vars/withMageEnv.groovy#L42

Reply:
will do separate PR with mage